docs/index.rst

```diff
@@ -78,7 +78,7 @@ compiler, and various Python packages.
 The simplest way to install Parcels is to use Anaconda and Conda-Forge for all 
 requirements. This is the “batteries-included” solution probably suitable for most users.
 
-1. Install Anaconda's Miniconda following the steps at https://conda.io/docs/user-guide/install/
+1. Install Anaconda's Miniconda following the steps at https://conda.io/docs/user-guide/install/. Be sure to download the Python 2.7 version.
```

Comment:
I'd rephrase this to:

> All the code below assumes that you download the Python-2 version.

Here's why: The Python version for the Parcels environment is specified in the `environment.yml`. Conda is perfectly fine with different Python versions in different envs. I usually have a Miniconda3 root env with different Python-2 and Python-3 environments for different use cases.